app/views/layouts/application.html.erb

```diff
@@ -23,6 +23,7 @@
                 <li class="nav-item"><%= link_to(t('nav.dashboard'), root_url, class: 'nav-link') %></li>
                 <li class="nav-item"><%= link_to(Issue.model_name.human(count: 2), issues_url, class: 'nav-link') %></li>
                 <li class="nav-item"><%= link_to(Job.model_name.human(count: 2), jobs_url, class: 'nav-link') if authorized?(:jobs) %></li>
+                <li class="nav-item"><%= text_field_tag :search_issue, nil, class: 'form-control' %></li>
```

Comment:
* Textfeld bitte in den Footer verschieben.
* Label: "Meldung bearbeiten"
* Placeholder: "Nummer"
* Feldlänge auf 6 Zeichen begrenzen, und nur Ziffern erlauben